_posts/2023-10-23-unidirectional-topic-operator.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: post
+title: "Unidirectional Topic Operator"
+date: 2023-10-23
+author: federico_valeri
+---
+
+The Topic Operator enhances Strimzi's capabilities by allowing Kafka administrators to manage Kafka topics with Kubernetes custom resources.
+You can declare the desired topic state and the Topic Operator will take care of the reconciliation with the actual Kafka cluster state.
+
+The first generation of the Topic Operator is based on the Kafka Streams library, and uses ZooKeeper to gather topic metadata.
+We call it **Bidirectional Topic Operator (BTO)**, as it allows to reconcile changes coming from both Kubernetes and Kafka using a three-way merge logic.
```

Comment:
I think it would be worth to elaborate on this by adding simple examples -> 
* you create or change a topic in Kube -> it is applied to Kafka
* you create to change a topic in Kafka -> it is applied to Kube

Reply:
I think it is clear if you continue to read few more lines where I talk about the UTO. I really don't want to give too much details about the 3way merge, as the main focus here is the new implementation.